.github/main.workflow

```diff
@@ -1,10 +0,0 @@
-workflow "Push" {
```

Comment:
Do not delete this `.github/main.workflow` file

Reply:
nice in theory, but git didn't work until I did that when I started work on that formula and I don't know if that bug is fixed by now...

Reply:
it made it impossible to push changes to my own fork

Reply:
Please undo changes to this file, squash everything into a single commit, and then we can debug problems if there are any. But all others PRs are working fine…

Reply:
I just checked, it seems like github fixed it on their end in the meantime